__tests__/ledger/LedgerArk.test.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const LedgerArk = require('../../lib/ledger/LedgerArk.js')
+
+describe('LedgerArk', () => {
+  it('should be defined', () => {
```

Comment:
Rather then simply expecting it to be defined we should expect it to be an object or an instance.

https://github.com/jest-community/jest-extended#tobeobject

Reply:
darn, should have seen that :) thanks!